init/sftpgo.service

```diff
@@ -17,6 +17,11 @@ KillMode=mixed
 PrivateTmp=true
 Restart=always
 RestartSec=10s
+NoNewPrivileges=yes
+PrivateDevices=yes
+DevicePolicy=closed
+ProtectSystem=true
+RestrictAddressFamilies=AF_INET AF_INET6
```

Comment:
please add `AF_UNIX`, the REST API can be configured to run over UNIX domain sockets

Reply:
Good point, done.